subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDeviceParser.kt

```diff
@@ -0,0 +1,26 @@
+package com.avito.runner.service.worker.device.adb
+
+class AdbDeviceParser {
+
+    fun parse(output: String): Set<AdbDeviceParams> {
```

Comment:
Do you think about reuse ddmlib or anything else?

Reply:
I know nothing about ddmlib and so on, just tried to dedicate & test parsing logic for AdbDevice